examples/makefile

```diff
@@ -54,6 +54,9 @@ endif
 ifeq ($(MFEM_USE_SUPERLU),YES)
    SUBDIRS += superlu
 endif
+ifeq ($(MFEM_USE_CALIPER),YES)
+   SUBDIRS += caliper
```

Comment:
Creating `examples/caliper` leads to duplication, and will get worse with increasing use of Caliper.